composeApp/src/commonMain/kotlin/com/arkivanov/minesweeper/game/GameContent.kt

```diff
@@ -62,15 +54,34 @@ internal fun GameContent(component: GameComponent, modifier: Modifier = Modifier
     val gridHeight by derivedStateOf { state.height }
     val grid by derivedStateOf { state.grid }
 
+    val remainingBombs by derivedStateOf {
+        state.maxMines - grid.values.count { it.status.isFlagged }
```

Comment:
Let's make it an extension, something like `val GameState.remainingMines: Int get() = ...`, and put it in `GameState.kt` file. The name `remainingMines` would be consistent with `maxMines`.